index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html> 
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/style.css">
+    <title> Index Document </title>
+</head>
+<body>
+      <nav>
+          <ul class="nav-list">
+            <li><a href="http://127.0.0.1:5500/Home.html"> Home </a></li>
+            <li><a href="http://127.0.0.1:5500/about.html"> About </a></li>
+            <li><a href="http://127.0.0.1:5500/portfolio.html"> Portfolio </a></li>
+          </ul>
+        </nav>
+        <header> 
```

Comment:
Consider wrapping the header all the way around the nav. Consulting MDN can give tips about how to apply tags.
>The `<header>` [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents introductory content, typically a group of introductory or navigational aids. It may contain some heading elements but also a logo, a search form, an author name, and other elements.